typescript-sdk/integrations/pydantic-ai/examples/README.md

```diff
@@ -0,0 +1,36 @@
+# Pydantic AI AG-UI Examples
+
+This directory contains example usage of the AG-UI adapter for Pydantic AI. It provides a FastAPI application that demonstrates how to use the Pydantic AI agent with the AG-UI protocol.
+
+## Features
+
+The examples include implementations for each of the AG-UI dojo features:
+- Agentic Chat
+- Human in the Loop
+- Agentic Generative UI
+- Tool Based Generative UI
+- Shared State
+- Predictive State Updates
+
+## Setup
+
+1. Install dependencies:
+   ```bash
+   uv sync
+   ```
+
+2. Run the development server:
+   ```bash
+   uv run dev
+   ```
+
+## Usage
+
+Once the server is running, you can access the different examples at:
+
+- `http://localhost:9000/agentic_chat` - Agentic Chat
+- `http://localhost:9000/agentic_generative_ui` - Agentic Generative UI
+- `http://localhost:9000/human_in_the_loop` - Human in the Loop
+- `http://localhost:9000/predictive_state_updates` - Predictive State Updates
+- `http://localhost:9000/shared_state` - Shared State
+- `http://localhost:9000/tool_based_generative_ui` - Tool Based Generative UI
```

Comment:
These are just the AG-UI endpoints though right? Wouldn't we want to point them to the Dojo app?

Reply:
Good call out. I left the note on these in there for people trying to play with the ag-ui endpoints directly but I called out running the dojo before that.